pipelines/push-disk-images-to-cdn/push-disk-images-to-cdn.yaml

```diff
@@ -191,6 +191,8 @@ spec:
       params:
         - name: dataPath
           value: "$(tasks.collect-data.results.data)"
+        - name: schema 
+          value: "https://raw.githubusercontent.com/konflux-ci/release-service-catalog/refs/heads/production/schema/dataKeys.json"
```

Comment:
I think instead of hardcoding production, you want taskGitRevision

Reply:
Although it may not make much difference. I don't see any easy way to use the taskGitUrl (since we want the raw version of the file) so e2e won't use the fork version anyways